react/pages/profile/ProfilePage/components/ProfileViews/index.js

```diff
@@ -55,7 +55,7 @@ const Channels = ({
 }) => (
   <EmptyMessageOrComponent
     identifiable={identifiable}
-    count={identifiable.counts.channels}
+    count={identifiable.counts.channels - 1}
```

Comment:
Why are we subtracting 1 again?

Reply:
Sorry, leftover, will remove.

Reply:
I guess this component can just handle this instead of it being a prop, also?